core/src/edu/asu/c3simulator/screens/FundingScreen.java

```diff
@@ -49,29 +50,27 @@ public void clicked(InputEvent event, float x, float y)
 				- (0.01f * stage.getWidth()), (0.01f * stage.getWidth()));
 		
 		// TODO add Corner Advisor
-		// TODO add Home Button
+		HomeButton home = new HomeButton(game);
 		
-		NavigationPanel navigationPanel = createNavigationPanel();
-		
-		stage.addActor(navigationPanel);
+		stage.addActor(home);
 		stage.addActor(continueButton);
 		stage.addActor(ScreenDiscription);
 	}
 	
 	@Override
-	public NavigationPanel createNavigationPanel()
+	public void createNavigationPanel()
```

Comment:
1: This comment was made in reply to **all navigation panels take some AllManagementScreens instance** in the context of **Alternatively, the navigationPanel could take an AllManagementScreens instance (e.g., AllManagementScreens.INDUSTRY) as an argument, instead of a Screen, and then direct game to screen.getInstance();**

2: Most lines were changed, except for MainHub, which has a separate cast, and calls instantiate() (line 67 of AllManagementScreens).

2.1: The idea is to generalize it to the point that we can enumerate all screens, and ensure a single instance of every screen. In order to do that, they all need to follow a single interface, with methods that apply to all screens. I think "initialize" is more generalized than "createNavigationPanel" and has the added benefit of being a common design pattern.